### ISSUE TITLE
pip refuse to install 1.0+ with python 3.7 without option --ignore-requires-python

### ISSUE THREAD

**Issue by juliendubost** [yunojuno/django-onfido#30]
```
pip install git+git://github.com/yunojuno/django-onfido@v1.1
```
```
ERROR: Package 'django-onfido' requires a different Python: 3.7.3 not in '<4.0,>=3.8'
``` 

Probably a packaging issue since v1.0 tests are OK with python3.7.3.

```
collected 54 items                                                                                                                         

tests/test_admin.py .                                                                                                                [  1%]
tests/test_api.py ....                                                                                                               [  9%]
tests/test_decorators.py ...                                                                                                         [ 14%]
tests/test_helpers.py ...                                                                                                            [ 20%]
tests/test_integration.py .                                                                                                          [ 22%]
tests/test_settings.py ....                                                                                                          [ 29%]
tests/test_views.py .                                                                                                                [ 31%]
tests/test_models/test_applicant.py ....                                                                                             [ 38%]
tests/test_models/test_base.py ..................                                                                                    [ 72%]
tests/test_models/test_check.py ...                                                                                                  [ 77%]
tests/test_models/test_event.py .....                                                                                                [ 87%]
tests/test_models/test_report.py ...                                                                                                 [ 92%]
tests/test_admin.py ...                                                                                                              [ 98%]
tests/test_data.py .                                                                                                                 [100%]
```

**Comment by juliendubost** [comment]
Ok so, after i installed from VCS
```
pip install git+git://github.com/yunojuno/django-onfido@v1.1
```
version 1.1 is compatible with python v3.8+ (in README it's 3.7+)

```
ERROR: Package 'django-onfido' requires a different Python: 3.7.3 not in '<4.0,>=3.8'
```

**Comment by hugorodgerbrown** [comment]
Hi @juliendubost - that was my fault - we updated a bunch of projects to support 3.8 (for the `:=` operator), and I updated the `pyproject.toml` and forgot about the classifiers. We don't actually use the walrus operator in this project, so I have updated the PEP517 config and re-released as 1.1.1. It should install on 3.7.

FYI the CI build only runs Py3.9 and Django 3.1, 3.2 and main, although the tox.ini does support 3.7-3.9 and Django 2.2-3.2.
